src/OrchardCore.Modules/OrchardCore.ContentLocalization/Shortcodes/LocaleShortcodeProvider.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Globalization;
+using System.Threading.Tasks;
+using Shortcodes;
+
+namespace OrchardCore.ContentLocalization.Shortcodes
+{
+    public class LocaleShortcodeProvider : IShortcodeProvider
```

Comment:
Why is it in `ContntLocalization`? I don't see the link with it.

Reply:
I was not sure where to put this shortcode. Do you have any suggestions?

Reply:
I think it's fine to be placed in `ContentLocalization` while the shortcode is related localize content

Reply:
It has nothing to do with Content, I don't understand where you get this idea from ;) It is related to current culture, and "text". Could either be in a Localization module or directly in the Shortcodes one.

Reply:
The "text" that need to be localized is the content itself

> Could either be in a Localization module 

I don't think so, the localization care about UI, dates, number, currencies .. etc, I think what @jptissot is suggest right, unless you see something I didn't aware of ;)

Reply:
Then shortcodes module. I don't even think it requires a reference to Localization.

Reply:
If this will go to `Shortcodes` no need to localization at all. At the beginning I thought it will localize the content, but after I dig into the shortcode and usage seems it doesn't make sense to be in content localization or localization module at all. You are right Seb ;) I revoke my previous words ;)

Reply:
@jptissot I'm with Seb, please move it to Shorcodes modules and remove all the localization references, while its not needed at all

Reply:
If it goes into shortcodes it’s fine.
But it needs RequiresFeature on its startup, possibly on localization.

It shouldn’t show up in the popup on single culture sites

Reply:
Ok for shortcode feature.